types/src/transaction.rs

```diff
@@ -526,6 +526,24 @@ impl IntoProto for SignedTransaction {
     }
 }
 
+impl TryFrom<crate::proto::types::SignedTransaction> for SignedTransaction {
+    type Error = Error;
+
+    fn try_from(txn: crate::proto::types::SignedTransaction) -> Result<Self> {
+        let signed_txn = SimpleDeserializer::deserialize(&txn.signed_txn)?;
+        Ok(signed_txn)
```

Comment:
I think you can just return the `deserialize` result directly here.

Reply:
yep you're right